kolena/_experimental/object_detection/workflow.py

```diff
@@ -197,12 +194,12 @@ class ThresholdConfiguration(EvaluatorConfiguration):
     Specify a confidence and IoU threshold to apply to all classes.
     """
 
-    threshold_strategy: Union[Literal["F1-Optimal"], float]
-    """The confidence threshold strategy. It can either be `F1_OPTIMAL="F1-Optimal"` or a fixed confidence threshold
-    such as `0.3` or `0.75`."""
+    threshold_strategy: Union[Literal["F1-optimal"], float] = "F1-optimal"
```

Comment:
I gotta say, `F1-optimal` looks odd after seeing so much `F1-Optimal` -- was this discussed already?

Reply:
Title case-ish is fine, I'll change it back 👍